File: 6. Remove characters from a string except alphabets/code.cpp
#include <iostream>
#include <string>
using namespace std;

string removeNonAlphabets(string s) {
    string result = "";
    for (char c : s) {
        if (isalpha(c)) {
            result += c;
        }
    }
    return result;
}

int main() {
    string input_str;
    cout << "Enter a string: ";
    getline(cin, input_str);
    cout << "String after removing non-alphabetic characters: " << removeNonAlphabets(input_str) << endl;
    return 0;
}

File: 6. Remove characters from a string except alphabets/code.java
import java.util.Scanner;

public class Main {
    public static String removeNonAlphabets(String s) {
        StringBuilder result = new StringBuilder();
        for (char c : s.toCharArray()) {
            if (Character.isLetter(c)) {
                result.append(c);
            }
        }
        return result.toString();
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String inputStr = scanner.nextLine();
        System.out.println("String after removing non-alphabetic characters: " + removeNonAlphabets(inputStr));
    }
}
